code/modules/mob/living/carbon/human/human_helpers.dm

```diff
@@ -85,8 +85,14 @@
 		. = if_no_id	//to prevent null-names making the mob unclickable
 	return
 
-//gets ID card object from special clothes slot or null.
-/mob/living/carbon/human/get_idcard()
+//Gets ID card object. If hand_first is false the one in the id slot is prioritized.
+/mob/living/carbon/human/get_idcard(hand_first = TRUE)
+	var/obj/item/I = get_active_held_item()

```

Comment:
expand this to check your PDA slot and your offhand too

order should prolly be ID slot -> PDA slot -> Belt slot -> Active -> Offhand -> Pocket

Reply:
isn't the id slot and pda slot the same?

Reply:
yeah there's no PDA slot here